feat(v2): add optional custom wrapper class name for pages based on theme classic Layout

Simek opened the pull request:
## Motivation

Some custom pages may require different layout or additional customizations to the main page wrapper.

This small PR introduces an ability to specify an optional, custom `wrapperClassName` to the theme classic `Layout` component.

I have **NOT** updated the docs because I was not able to find the page which describes all other optional `Layout` props. ["Theme classic"](https://v2.docusaurus.io/docs/theme-classic) and ["Creating pages"](https://v2.docusaurus.io/docs/creating-pages) only briefly mention `Layout` usage. The most properties is used in the ["Migration guide"](https://v2.docusaurus.io/docs/migrating-from-v1-to-v2#pages) but still without a description.

I can add the note about this feature if you will be able to tell me where to put it. Alternatively, I can (in a later time) try to add `Layout` component related section to the Theme Classic page. 🙂 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Feature has been tested with the local Docusaurus V2 website.
<img width="367" alt="Screenshot 2020-09-16 124227" src="https://user-images.githubusercontent.com/719641/93327524-c370a900-f81a-11ea-9123-d6ec8383aa1d.png">

## Related PRs

* react-native-website-migration/react-native-website#23

lex111 reviewed (commented):
This is nice feature indeed, I think what else can we add a similar feature for MD page in its frontmatter?

```
---
title: my hello page title
description: my hello page description
wrapperClassName: hello
---

Content..
```

> Alternatively, I can (in a later time) try to add Layout component related section to the Theme Classic page.

That would be great, we definitely  need  it.

Simek commented:
@lex111 I have added the `wrapperClassName` property for Markdown pages and updated the Markdown page example within the V2 website repository:
* https://v2.docusaurus.io/examples/markdownpageexample/

![image](https://user-images.githubusercontent.com/719641/93334064-94f7cb80-f824-11ea-9b17-80d9b73e3a7a.png)

Simek commented:
@lex111 Is there anything more you would like to see implemented within this PR? 🙂

lex111 reviewed (approved):
LGTM, we will write docs for Layout component in a separate PR.

lex111 commented:
@Simek thank you!

Simek commented:
> LGTM, we will write docs for Layout component in a separate PR.

@lex111 I'm happy to do this after I finish the RN website migration.
